.prettierignore

```diff
@@ -0,0 +1,2 @@
+src/__generated__/*.ts
+node_modules
```

Comment:
also, a great move 👍

Reply:
Turns out that the graphql-js patch wasn’t applying because the whole file had changed due to prettier on save :)